workflows/scrnaseq/10x-bam-to-fastqs.wdl

```diff
@@ -15,7 +15,7 @@
 ## : an array of files sufficient for localizing in Cell Ranger's expected format
 ##
 ## fastqs_archive
-## : a compressed archive comtaining the array of FastQ files
+## : a compressed archive comtaining the array of FASTQ files
```

Comment:
```suggestion
## : a compressed archive containing the array of FASTQ files
```

Not related to this PR, but since you made a change on this line anyway.